### PR TITLE
feat(combo): add single-selection option

### LINKED ISSUE #626: Single Selection Combo
_Estimated Delivery Sprint - Sprint X_

- [x] Research
- [x] Specification
- [x] Component and tests implementation
- [x] API Documentation
- [x] Design implementation

### PR DESCRIPTION
**Pull request by simeonoff**
Closes #626

### CODE REVIEW

**Comment by SisIvanova** [comment]
We should remove the text highlight when the combo is disabled


https://user-images.githubusercontent.com/59446295/213431568-9df458fa-202f-4b1c-83d0-92306b58306b.mov

**Review by SisIvanova — APPROVED** [review]

File: src/components/combo/combo-item.ts [hunk 1]
```diff
@@ -31,6 +31,9 @@ export default class IgcComboItemComponent extends LitElement {
   @property({ type: Boolean, reflect: true })
   public active = false;
 
+  @property({ attribute: 'hide-checkbox', type: Boolean, reflect: true })
```
Comment (rkaraivanov) [inline src/components/combo/combo-item.ts:34]:
```suggestion
  @property({ attribute: false, type: Boolean })
```
Unless there is a styling need, I don't see a reason to have a reflecting attribute here

File: src/components/combo/combo-item.ts [hunk 2]
```diff
@@ -51,6 +54,7 @@ export default class IgcComboItemComponent extends LitElement {
           ?checked=${this.selected}
           tabindex="-1"
           exportparts="control: checkbox, indicator: checkbox-indicator, checked"
+          ?hidden=${this.hideCheckbox}
```
Comment (rkaraivanov) [inline src/components/combo/combo-item.ts:57]: Do we really need to render a hidden checkbox when single selection is enabled?

**Review by rkaraivanov — CHANGES_REQUESTED** [review]
LGTM

**Review by rkaraivanov — APPROVED** [review]

**Review by SisIvanova — APPROVED** [review]

**Pull request merged**